### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bartcoelus**
The link to join the community on Github resulted in a 404 page - fixed
Added the Linkedin group - a share and learn group 
Added some missing dots

### CODE REVIEW

**Comment by Stebalien** [comment]
This is the first I've heard of the linked-in community. Given that we have a discussion forum, I'd rather not list yet another discussion medium; especially one on a closed platform like linked-in.

@flyingzumwalt, is this a community you're aware of?

File: README.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ If the issue is a protocol weakness that cannot be immediately exploited or some
 
 ## Project and Community
 
-[![](https://cdn.rawgit.com/jbenet/contribute-ipfs-gif/master/img/contribute.gif)](https://github.com/ipfs/community/blob/master/CONTRIBUTE.md)
+[![](https://cdn.rawgit.com/jbenet/contribute-ipfs-gif/master/img/contribute.gif)](https://github.com/ipfs/community)
```
Comment (Stebalien) [inline README.md]: Should be https://github.com/ipfs/community/blob/master/CONTRIBUTING.md.

File: README.md [hunk 2]
```diff
@@ -252,26 +252,28 @@ There are two deprecated repositories containing [FAQ](https://github.com/ipfs/f
...
 - [distributions](https://github.com/ipfs/distributions) - Scripts to build the /install html page.
-- [newsletter](https://github.com/ipfs/newsletter) - Prepare and store IPFS Newsletter roundups
+- [newsletter](https://github.com/ipfs/newsletter) - Prepare and store IPFS Newsletter roundups.
 - [infrastructure](https://github.com/ipfs/infrastructure) - Tools and systems for the community.
-- [ops-requests](https://github.com/ipfs/ops-requests) - Requests about infrastructure operations
-- [project-repos](https://github.com/ipfs/project-repos) CI status and other health metrics
+- [ops-requests](https://github.com/ipfs/ops-requests) - Requests about infrastructure operations.
+- [project-repos](https://github.com/ipfs/project-repos) CI status and other health metrics.
 - [website](https://github.com/ipfs/website) - The source to the IPFS community website at http://ipfs.io.
+- [IPFS group on Linkedin](https://www.linkedin.com/groups/13635533/) - Open group to share and learn about IPFS on Linkedin.
```
Comment (lidel) [inline README.md]: We don't link to Reddit ([/r/ipfs/](https://www.reddit.com/r/ipfs/)), so makes little sense to add Linked-in. 
AFAIK as a rule of thumb, we don't link  to communities at third party websites outside of our official infrastructure and control as it gets nasty really quick: what are criteria for "endorsing" a community? how do we control abuse and moderation? not to mention confusing and splitting community into tiny silos.

I vote for removing this line:

```suggestion

```

**Comment by raulk** [comment]
In my experience, LinkedIn groups are vastly ineffective to discuss technical topics. Often they’re used for announcements and marketing by corporations. We already have the IPFS Discourse, and fragmenting the community is not something I’d advise doing.

**Comment by Stebalien** [comment]
I'm mostly wary of having an *endorsed* community without anyone with a solid technical understanding of IPFS. There are, unfortunately, quite a few misconceptions about what IPFS is, what it does, and how it relates to blockchain tech given all the buzz in this space.

That doesn't mean the community shouldn't exist; networking around a shared interest in IPFS is great! I'm just reluctant to *direct* people to a community where we can't vouch for the accuracy of information therein (and can't even see it without an account).

**Comment by bartcoelus** [comment]
I understand - but it's there and I taught the purpose was to inform the
people of all the groups and communities there are. Let's just remove that
Linkedin line then... it's mentioned from time to time on Reddit anyhow.
Thanks for considering everything so precise, that's very  professional.

File: README.md (hunk 2 above)
Comment (Stebalien) [reply README.md]: That sounds totally reasonable. Each newsletter is constructed here: https://github.com/ipfs/newsletter/pulls.

Comment (spyhole) [reply README.md]: Linkedin group , wait there, i m coming~~~

**Comment by spyhole** [comment]
GREAT WORK!

**Comment by spyhole** [comment]
In my experience, LinkedIn groups are vastly ineffective to discuss technical topics. Often they’re used for announcements and marketing by corporations. +1

**Comment by Stebalien** [comment]
> In my experience, LinkedIn groups are vastly ineffective to discuss technical topics. Often they’re used for announcements and marketing by corporations. +1

The idea of the LinkedIn group *wasn't* to encourage technical discussions, it was to allow people with a common interest in IPFS to "network".

**Pull request merged**